docs/attributes-registry/messaging.md

```diff
@@ -51,6 +51,8 @@ size should be used.
 
 **[7]:** If a custom value is used, it MUST be of low cardinality.
 
+**[8]:** Messaging instrumentations MUST use applicable value from the [list of known messaging systems](/docs/messaging/README.md#technology-specific-semantic-conventions) when it is available. When instrumenting a new messaging system, instrumentation authors should document the new system name and add it to the [list](/docs/messaging/README.md#technology-specific-semantic-conventions).
```

Comment:
New identifiers for messaging systems shouldn't be added to the README, but to the YAML files.